### PR TITLE
Deppy Framework (5/7): root deppy command

### PR DESCRIPTION
**Pull request by perdasilva**
This PR adds the root deppy command and associated make target

### CODE REVIEW

**Review by oceanc80 — APPROVED** [review]

File: cmd/root/root.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+var rootCmd = &cobra.Command{
+	Use:   "deppy",
+	Short: "Deppy is an open-source constraint solver framework",
+	Long: `An open-source constraint solver framework written in Go.
+For more information visit https://github.com/operator-framework/deppy`,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		_, _ = fmt.Fprintln(os.Stderr, err)
```
Comment (timflannagan) [inline cmd/root/root.go:19]: Any reason why we can't just do `fmt.Fprintln(os.Stderr, err)` here without capturing any of the return values?

File: cmd/root/root.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package root
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
```
Comment (timflannagan) [inline cmd/root/root.go:10]: nit: Use a constructor that returns a *cobra.Command vs. a packge-level variable?

File: cmd/root/root.go (hunk 1 above)
Comment (perdasilva) [reply cmd/root/root.go]: I just get annoyed by the squiggles in the ide...this was just to print to stderr btw

File: cmd/root/root.go (hunk 2 above)
Comment (perdasilva) [reply cmd/root/root.go]: I've done it. But, how come?

Comment (joelanford) [reply cmd/root/root.go]: generally in my experience, globals variables are headaches

**Pull request merged**